src/components/Organizations/Detail/View.js

```diff
@@ -86,6 +97,11 @@ class OrganizationDetail extends React.Component {
           return cmp(a, b);
         },
       },
+      {
+        dataField: 'path',
+        text: '',
+        formatter: upgradeNoticeIcon.bind(this),
```

Comment:
Can you please use an arrow function here, so we could get rid of using `this` inside a function?

Reply:
I think it won't work as the function is declared outside the class

Reply:
What I'm suggesting is to reach an implementation similarto:
```javascript
  formatter: (_, cluster) =>  upgradeNoticeIcon(cluster, ...someOtherProps)
```
+ modifying `upgradeNoticeIcon` accordingly

So we wouldn't have to use `this` inside the `upgradeNoticeIcon`.